sklearn/model_selection/_validation.py

```diff
@@ -781,6 +781,23 @@ def _score(estimator, X_test, y_test, scorer, error_score="raise"):
                 UserWarning,
             )
 
+    # Check errors in `_MultimetricScorer`
+    if isinstance(scorer, _MultimetricScorer):
+        exceptions = {name: e for name, e in scores.items() if isinstance(e, Exception)}
+        if exceptions:
+            if error_score == "raise":
+                for name, e in exceptions.items():
+                    raise e
```

Comment:
Im not sure if this is the best way to access an element of the dict. Actually I think it doesnt matter which exception we choose here.

Reply:
We can use a list instead:

```python
exceptions = [(name, e) for name, e in scores.items() if isinstance(e, Exception)]
```

This way we can raise directly here:

```python
raise exceptions[0][1]
```

`new_scores` would need to be updated slightly.